station_1: [0.5,0.5,0.0]
station_2: [0.5,-0.5,0.0]
station_3: [-0.5,0.5,0.0]
station_4: [-0.5,-0.5,0.0]
station_5: [0.0,0.0,0.0]


dis_1: [0.0,1.0,0.0,0.0,0.7] 
dis_2: [1.0,0.0,0.0,0.0,0.7] 
dis_3: [0.0,0.0,0.0,1.0,0.7] 
dis_4: [0.0,0.0,1.0,0.0,0.7] 
dis_5: [0.7,0.7,0.7,0.7,0.0] 

